client/state/comments/actions.js

```diff
@@ -160,6 +161,11 @@ export function requestPostComments( siteId, postId, status = 'approved' ) {
 	};
 }
 
+export const requestCommentsList = query => ( {
+	type: COMMENTS_LIST_REQUEST,
```

Comment:
@gwwar I haven't been in there for a while but I think more things are messed up. I get about six to nine instances the dev tools want to hook into. I do remember seeing this action though when I dispatched from the console